app/src/test/java/org/mozilla/focus/ext/ContextTest.kt

```diff
@@ -0,0 +1,53 @@
+package org.mozilla.focus.ext
+
+import android.content.Context
+import android.view.accessibility.AccessibilityManager
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.Shadows.shadowOf
+import org.robolectric.util.ReflectionHelpers
+
+@RunWith(RobolectricTestRunner::class)
+class ContextTest {
+
+    private var context: Context? = null
+    private var accessibilityManager: AccessibilityManager? = null
+
+    @Before
+    fun startUp() {
+        context = RuntimeEnvironment.application
+        accessibilityManager = context!!.getAccessibilityManager()
+    }
+
+    @Test
+    fun `isScreenReaderEnabled() returns true when TouchExploration enabled`() {
```

Comment:
Nitpick: Shouldn't this method be called `isVoiceViewEnabled() returns true when TouchExploration enabled`?

Reply:
Yes. It should be. It was picked from a commit from android-components and I forgot to update method names accordingly